be/src/exec/hdfs_scanner.cpp

```diff
@@ -105,6 +104,16 @@ Status HdfsScanner::_build_scanner_context() {
         partition_values.emplace_back(std::move(partition_value_column));
     }
 
+    // evaluate extended column values
+    std::vector<ColumnPtr>& extended_values = ctx.extended_values;
+    for (size_t i = 0; i < _scanner_params.extended_col_slots.size(); i++) {
+        int extended_col_idx = _scanner_params.index_in_extended_columns[i];
+        ASSIGN_OR_RETURN(auto extended_value_column,
```

Comment:
Is's this expression evaluated in `hive_connector.cpp`?

Reply:
done